Tutorial.ipynb

```diff
@@ -1,1105 +1,532 @@
-     "exception": false,
-     "start_time": "2022-04-19T18:37:32.750775",
-     "status": "completed"
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bc55499c"
+      },
+      "source": [
+        "<h3> Here we read in our validation data, then pre-process our input data from this and make predictions (showing two different ways to do this). Finally, we pre-process our existing labels to be able to calculate performance metrics </h3>"
+      ],
+      "id": "bc55499c"
     },
-    "tags": []
-   },
-   "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "[[838   4]\n",
-      " [121  79]]\n",
-      "              precision    recall  f1-score   support\n",
-      "\n",
-      "       False       0.87      1.00      0.93       842\n",
-      "        True       0.95      0.40      0.56       200\n",
-      "\n",
-      "    accuracy                           0.88      1042\n",
-      "   macro avg       0.91      0.70      0.74      1042\n",
-      "weighted avg       0.89      0.88      0.86      1042\n",
-      "\n",
-      "0.5676294827683775\n"
-     ]
-    }
-   ],
-   "source": [
-    "# Default threshold is 0.5. I've found you need to go over 0.9 to start really seeing an effect\n",
-    "threshold=0.994\n",
-    "conf_predictions=(predictor.predict_proba(texts)[:,1] >= threshold).astype(bool)\n",
-    "\n",
-    "print(confusion_matrix((y_val),conf_predictions))\n",
-    "print(classification_report((y_val), conf_predictions))\n",
-    "print(matthews_corrcoef((y_val), conf_predictions))"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "1f3d7866",
-   "metadata": {
-    "papermill": {
-     "duration": 0.014828,
-     "end_time": "2022-04-19T18:37:56.707253",
-     "exception": false,
-     "start_time": "2022-04-19T18:37:56.692425",
-     "status": "completed"
+      "cell_type": "code",
+      "execution_count": 14,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "1918321c",
+        "outputId": "a231c36d-e8f3-400f-c73a-9e68236a3172"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "CPU times: user 31.6 s, sys: 5.29 s, total: 36.8 s\n",
+            "Wall time: 3min 11s\n"
+          ]
+        }
+      ],
+      "source": [
+        "# Unzip and read in validation data\n",
+        "import shutil\n",
+        "shutil.unpack_archive('../pathway-abstract-classifier/Data/val_data.tsv.zip', '../pathway-abstract-classifier/Data')\n",
+        "\n",
+        "df=pd.read_csv('../pathway-abstract-classifier/Data/val_data.tsv', delimiter=\"\\t\")\n",
+        "\n",
+        "#pre-process input data and make predictions (method 1 - using lists)\n",
+        "titles=df['title'].tolist()\n",
+        "abstracts=df['abstract'].tolist()\n",
+        "sep_token = predictor.preproc.get_tokenizer().sep_token\n",
```

Comment:
I wouldn't flip between `sep_token` and `"[SEP]"`, that's confusing. I would use one or the other. If you feel this needs to be explained to the user that's OK. You could have a code cell that sets up `sep_token`. You could explain in plain English what you are doing and why and then use `sep_token` throughout (not the literal `"[SEP]"`).